crates/gen/default/readme.md

```diff
@@ -3,7 +3,7 @@ dependent crate or workspace has an empty or non-existent `.windows/winmd` direc
 
 ## Windows.Win32.winmd
 - Source: https://www.nuget.org/packages/Microsoft.Windows.SDK.Win32Metadata/
-- Version: 10.0.19041.5-preview.166
+- Version: 10.0.19041.202-preview
```

Comment:
@kennykerr I wasn't really sure what to do with the version. The previous `.166` release used this "new" format too, is this correct?

Reply:
Yes, I basically just copy the format from the win32metadata release verbatim.